### PR TITLE
update install instructions with proper path

### PR DESCRIPTION
**Pull request by zgoldman-r7**
Updates readme with proper instructions for chocolatey install using the correct path

### CODE REVIEW

File: c/meterpreter/README.md [hunk 1]
```diff
@@ -35,42 +35,12 @@ Building with VS2019 works with any version, including community. If you have an
...
-3. Install VS with all the required components by running the following command in Powershell:
+2. Install VS with all the required components by running the following command in Powershell:
   ```
-choco install visualstudio2019community -y --package-parameters "--config Path\To\Your\.vsconfig"
+choco install visualstudio2019community -y --package-parameters "--config C:\YOUR_PATH\metasploit-payloads\c\meterpreter\vs-configs\vs2019.vsconfig"
   ```
 
-Note: A copy of this file is located in this repository under `c/meterpreter/vs-config/vs2019.vsconfig`.
-
 ### VS 2017
```
Comment (adfoster-r7) [inline c/meterpreter/README.md:44]: Looks like this would need the same changes for the 2017 section :+1:

Comment (zgoldman-r7) [reply c/meterpreter/README.md:44]: Good catch!

File: c/meterpreter/README.md [hunk 2]
```diff
@@ -82,23 +52,9 @@ Building with VS2017 works with any version, including community. If you have an
...
-        "microsoft.visualstudio.component.vc.atl",
-        "microsoft.visualstudio.componentgroup.nativedesktop.winxp"
-    ]
-}
-  ```
-3. Install VS with all the required components by running the following command in Powershell:
+2. Install VS with all the required components by running the following command in Powershell:
   ```
-choco install visualstudio2017community -y --package-parameters "--config Path\To\Your\.vsconfig"
+choco install visualstudio2019community -y --package-parameters "--config C:\YOUR_PATH\metasploit-payloads\c\meterpreter\vs-configs\vs2017.vsconfig"
```
Comment (adfoster-r7) [inline c/meterpreter/README.md:57]:
```suggestion
choco install visualstudio2017community -y --package-parameters "--config C:\YOUR_PATH\metasploit-payloads\c\meterpreter\vs-configs\vs2017.vsconfig"
```

Comment (zgoldman-r7) [reply c/meterpreter/README.md:57]: whoops, good catch

**Pull request merged**